tests/target/issue-2321/trailing_comma_vertical_and_brace_style_prefer_same_line.rs

```diff
@@ -0,0 +1,9 @@
+// rustfmt-brace_style: PreferSameLine
+// rustfmt-trailing_comma: Vertical
+
+fn lorem<S, T>(lorem: S, ipsum: T)
+where
+    S: Add + Sub,
+    T: Mul + Div {
```

Comment:
Looking at this output, I'd actually expect there to be a trailing comma after `T: Mul + Div`, since the where clause bounds were formatted vertically.